src/Components/Web/src/Forms/InputNumber.cs

```diff
@@ -14,9 +14,9 @@ namespace Microsoft.AspNetCore.Components.Forms
     /// </summary>
     public class InputNumber<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)] TValue> : InputBase<TValue>
     {
-        private readonly static string _stepAttributeValue; // Null by default, so only allows whole numbers as per HTML spec
+        private static readonly string _stepAttributeValue = GetStepAttributeValue();
```

Comment:
Are we breaking the assumption made by the code comment here? If so, is that safe to do?